receiver/awsecscontainermetricsreceiver/testdata/task_metadata2.json

```diff
@@ -0,0 +1,116 @@
+{
```

Comment:
Are these files referenced in tests?

Reply:
No, but these are some corner cases which I think it can be used in the further development. I can delete this if you think it is not reasonable.

Reply:
Well we need some unit tests for the new behavior, I thought these would be it?